src/Microsoft.ML.Data/Evaluators/RankerEvaluator.cs

```diff
@@ -806,7 +806,7 @@ protected override void UpdateState(RowCursorState state)
                 RankerUtils.QueryDcg(_labelGains, _truncationLevel, state.QueryLabels, state.QueryOutputs, state.DcgCur);
                 for (int t = 0; t < _truncationLevel; t++)
                 {
-                    Double ndcg = state.MaxDcgCur[t] > 0 ? state.DcgCur[t] / state.MaxDcgCur[t] * 100 : 0;
```

Comment:
>100 [](start = 98, length = 3)

[https://github.com/dotnet/machinelearning/blob/master/docs/images/NDCG.png](https://github.com/dotnet/machinelearning/blob/master/docs/images/NDCG.png)
you need to update this one :D

Reply:
I swear I did! :)

---
In reply to: [259520368](https://github.com/dotnet/machinelearning/pull/2697#discussion_r259520368) [](ancestors = 259520368)

Reply:
Do you mean change back to having the multiplier?

---
In reply to: [259520368](https://github.com/dotnet/machinelearning/pull/2697#discussion_r259520368) [](ancestors = 259520368)

Reply:
No, I mean we use this image in our documents. Would be strange if we change values to be from 0 to 1, but in documentation it would still be multiplied by 100

Reply:
Oh, good eyes. I'll update the docstrings too.

---
In reply to: [259548793](https://github.com/dotnet/machinelearning/pull/2697#discussion_r259548793) [](ancestors = 259548793)